### DISCUSSION TITLE
My first theme, Industrial

### DISCUSSION THREAD

**Discussion by wsokc** [mathoudebine/turing-smart-screen-python#567] [Themes]
This is my first theme, there are some more development needed especially on the storage section feel free to add
[Industrial.zip](https://github.com/user-attachments/files/16901923/Industrial.zip)
![preview](https://github.com/user-attachments/assets/821a65e6-107e-4df9-96e4-d617c6c9e5eb)

**Comment by DJRoby19** [comment]
Excellent work, unfortunately I always have the problem of the GPU fan which is not displayed in this version 3.5.0

**Reply by wsokc** [reply]
just realized, you are right, my gpu fan also not pickup

**Reply by DJRoby19** [reply]
I reported the problem as soon as version 3.5.0 was released, version 3.4.0 worked but no one ever replied to me
